<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>858</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Directed Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nodeNameTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Node Unique Name</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>16</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nodeHuerTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Node Heuristic. If not available, it will be considered 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPathCTxt">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Edge Path Cost. If not available, it will be considered 0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fromEdAddCom">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>From Node</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toEdAddCom">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>To Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adNodeBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adEdgeBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Edge</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="delNodeCom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick a Node to Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delNodeBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Node</string>
    </property>
   </widget>
   <widget class="QComboBox" name="delEdCom">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick an Edge to Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delEdgeBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>130</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Edge</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="startNodeCom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick a Starting Node</string>
    </property>
   </widget>
   <widget class="QComboBox" name="searchAlgoCom">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick a Searching Algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Goal Node</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
